dispatch-service/src/main/java/de/muenchen/oss/swim/dispatcher/configuration/SwimDispatcherProperties.java

```diff
@@ -73,6 +73,12 @@ public class SwimDispatcherProperties {
      */
     @NotEmpty
     private List<UseCase> useCases = List.of();
+    /**
+     * Metadata IndexFields key to use for destination binding.
+     * {@link UseCase#isDestinationViaMetadata()}
+     */
+    @NotBlank
+    private String metadataDispatchBindingKey;
```

Comment:
should this really be in the area of `// ########### use cases ###########`? Or rather be a separate section like `metadata`?